src/inspect_tool_support/src/inspect_tool_support/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Sandbox container tool code for inspect_ai.
+
+Contains tools for web browser, bash, and editor functionality.
+"""
+
+__version__ = "0.1.0"
```

Comment:
This seems bogus since it's listed in `pyproject.toml`. Hmmm.

Reply:
the pattern we use in Inspect for this is:

```
 __version__ = importlib_version(PKG_NAME)
```